frontend/src/features/properties/components/LastUpdatedBy.tsx

```diff
@@ -14,35 +14,43 @@ const UpdateText = styled.p`
   padding: 1rem;
   display: flex;
   justify-content: flex-end;
+  margin-bottom: 0px;
 `;
 
 const LinkButton = styled.span`
-  background: none;
-  border: none;
-  padding: 0;
-  color: #069;
+  color: #18548c;
   text-decoration: underline;
 `;
 
 /**
  * Display formatted last update user name, email, and date/time.
  */
-const LastUpdatedBy: React.FunctionComponent<ILastUpdatedByProps> = ({
+const LastUpdatedBy: React.FC<ILastUpdatedByProps> = ({
```

Comment:
😎

Reply:
I think that was an accident lol, I couldn't figure out what an error meant so I was just trying stuff